File: component-acgschedule/src/main/runalone/java/com/rabtman/acgschedule/AcgScheduleMainActivity.java
package com.rabtman.acgschedule;

import android.widget.FrameLayout;
import butterknife.BindView;
import com.rabtman.acgschedule.mvp.ui.fragment.ScheduleMainFragment;
import com.rabtman.common.base.SimpleActivity;

/**
 * @author Rabtman
 */

public class AcgScheduleMainActivity extends SimpleActivity {

  @BindView(R.id.layout_load_fragment)
  FrameLayout layoutLoadFragment;

  @Override
  protected int getLayoutId() {
    return R.layout.activity_runalone;
  }

  @Override
  protected void initData() {
    ScheduleMainFragment scheduleMainFragment = new ScheduleMainFragment();
    loadRootFragment(R.id.layout_load_fragment, scheduleMainFragment);
  }
}

File: component-acgschedule/src/main/runalone/java/com/rabtman/acgschedule/AcgScheduleMainActivity.kt
package com.rabtman.acgschedule

import android.widget.FrameLayout
import butterknife.BindView
import com.rabtman.acgschedule.mvp.ui.fragment.ScheduleMainFragment
import com.rabtman.common.base.SimpleActivity

/**
 * @author Rabtman
 */
class AcgScheduleMainActivity : SimpleActivity() {
    @BindView(R2.id.layout_load_fragment)
    lateinit var layoutLoadFragment: FrameLayout

    override fun getLayoutId(): Int {
        return R.layout.activity_runalone
    }

    override fun initData() {
        val scheduleMainFragment = ScheduleMainFragment()
        loadRootFragment(R.id.layout_load_fragment, scheduleMainFragment)
    }
}